src/main/java/net/minecraftforge/registries/DataSerializerEntry.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016-2018.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.registries;
+
+import net.minecraft.network.datasync.DataSerializer;
+
+public final class DataSerializerEntry extends IForgeRegistryEntry.Impl<DataSerializerEntry>
```

Comment:
Is the only reason for this class to hide the wildcard generic?

Reply:
`DataSerializer` is an interface (with anonymous implementations) - it can't easily be used as a registry type.